apps/src/studio/levels.js

```diff
@@ -2651,6 +2651,46 @@ levels.js_hoc2015_change_setting = {
   ]
 };
 
+var js_hoc2015_event_free_character_instructions = '"You\'re on your own now, BOT1."';
```

Comment:
i suspect we'll need to get instructions out of this file and into a translatable json file in the near future, and it's tracked on our to-do list.  just mentioning in case it'll be easier for you to do sooner than later.